### PR TITLE
#162342187 Tests the properties and relationships of profile model.

### PR DESCRIPTION
**Pull request by gbols**
#### What does this PR do?
It tests the relationships and properties of the profile model.
#### Description of Task to be completed?
create endpoint to use this model.
#### How should this be manually tested?
none 
#### Any background context you want to provide?
none
#### What are the relevant pivotal tracker stories?
#[162342187](https://www.pivotaltracker.com/story/show/162342187)
#### Screenshots (if appropriate)
none

### CODE REVIEW

File: tests/models/profile.test.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { expect } from 'chai';
+import { sequelize, dataTypes, checkModelName, checkPropertyExists}  from 'sequelize-test-helpers';
+import ProfileModel from '../../server/models/profile';
+import UserModel from '../../server/models/user';
+
+describe('Profile properties', () => {
+  const Profile = ProfileModel(sequelize, dataTypes);
+  const profile = new Profile();
+  checkModelName(Profile)('Profile');
```
Comment (V-84) [inline tests/models/profile.test.js:9]: You can use the it() block for better readability

File: tests/models/profile.test.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  context('properties', () => {
+       ['gender','twitterUrl','role','imageurl','facebookUrl','bio','dateOfBirth'].forEach(checkPropertyExists(profile));
+  });
+  context('check associations', () => {
+    const User = UserModel(sequelize, dataTypes);
+    before(() => {
+      Profile.associate({User});
+    })
+    it('defined a belongsTo association with UserModel', () => {
```
Comment (V-84) [inline tests/models/profile.test.js]: it description text should begin with should

File: tests/models/profile.test.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      expect(Profile.belongsTo.calledWith(User)).to.equal(true);
+    });
+  });
+});
```
Comment (V-84) [inline tests/models/profile.test.js]: no newline at the end of file

File: tests/models/profile.test.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+import { sequelize, dataTypes, checkModelName, checkPropertyExists}  from 'sequelize-test-helpers';
+import ProfileModel from '../../server/models/profile';
+import UserModel from '../../server/models/user';
+
+describe('Profile properties', () => {
+  const Profile = ProfileModel(sequelize, dataTypes);
+  const profile = new Profile();
+  checkModelName(Profile)('Profile');
+  context('properties', () => {
+       ['gender','twitterUrl','role','imageurl','facebookUrl','bio','dateOfBirth'].forEach(checkPropertyExists(profile));
```
Comment (V-84) [inline tests/models/profile.test.js:11]: nice block of code.
but I think using it() makes it more readable

**Review by V-84 — COMMENTED** [review]
nice work gbols. But please take a look at the reviews to make this better

File: tests/models/profile.test.js (hunk 1 above)
Comment (gbols) [reply tests/models/profile.test.js:9]: it is already readable as it is, after the test is run a comment that the model exits is displayed on the console.

File: tests/models/profile.test.js (hunk 4 above)
Comment (gbols) [reply tests/models/profile.test.js:11]: That would require writing the same line of code 9 times which i think is verbose.
```
has property gender
```
is descriptive enough.

**Review by efalayi — APPROVED** [review]
LGTM

**Pull request merged**